### PR TITLE
Business template

### PR DESCRIPTION
**Pull request by FabioSilvaOS**
I created this Business Template using Sqlite as database working integrated with Asp.Net Membership. Because it is based in the 4_6 branch, Github is asking to merge against it, but it can be useful keep both versions because both are different samples projects (the original is about a students managements)

### CODE REVIEW

File: src/SilverlightWCFRIA/Generated_Code/SilverlightWCFRIA.Web.g.cs
Comment (jacob-l) [inline src/SilverlightWCFRIA/Generated_Code/SilverlightWCFRIA.Web.g.cs:1]: Can we remove the generated code from the repo and add adjust .gitignore?

File: src/SilverlightWCFRIA/SilverlightWCFRIA.csproj [hunk 1]
```diff
@@ -83,6 +93,12 @@
 
   <ItemGroup>
     <Folder Include="Generated_Code\" />
+    <Folder Include="Models\" />
+    <Folder Include="Helpers\" />
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <UpToDateCheckInput Remove="Views\About.xaml" />
```
Comment (jacob-l) [inline src/SilverlightWCFRIA/SilverlightWCFRIA.csproj]: We do not need this. That is a strange behavior of Visual Studio. It constantly creates this config.

**Review by jacob-l — CHANGES_REQUESTED** [review]
Hi @FabioSilvaOS , thank you very much. The app is wonderful!

I have tried to run it locally. I have some issues with the backend side:
Снимок экрана 2023-09-15 в 14 06 28

Do you know what is the reason? I have tried to disable this module, however, it looks like it is used because I can not create a user without this module :)

And can we also check the error handling code:
Снимок экрана 2023-09-15 в 14 31 32

Thanks!

**Comment by jacob-l** [comment]
Hi @FabioSilvaOS , Thank you for your fixes! It works much better now.
I paid attention to some files in the SilverlightWCFRIA.Web project: ClientBin\SilverlightWCFRIA.xap, SilverlightWCFRIATestPage.aspx, SilverlightWCFRIATestPage.html, Silverlight.js.
I think we do not need these files. Or do I miss something?

And do we require database.db file in the repository?

Also, can we show a simple HTML page when the user tries to run the server-side project? Something basic like:
"That is a backend-side project. To access the app's interface, you should run the Browser project."

Thanks!

**Comment by jacob-l** [comment]
Hi @FabioSilvaOS , we had a chat with Giovanni. He suggests adding something like To-Do items / Tasks to this example. So the user can register, login, and then do something else: see/manage his tasks. We would like to get a more solid Business Template example. What are your thoughts?

**Comment by FabioSilvaOS** [comment]
> Hi @FabioSilvaOS , Thank you for your fixes! It works much better now. I paid attention to some files in the SilverlightWCFRIA.Web project: ClientBin\SilverlightWCFRIA.xap, SilverlightWCFRIATestPage.aspx, SilverlightWCFRIATestPage.html, Silverlight.js. I think we do not need these files. Or do I miss something?
> 
> And do we require database.db file in the repository?
> 
> Also, can we show a simple HTML page when the user tries to run the server-side project? Something basic like: "That is a backend-side project. To access the app's interface, you should run the Browser project."
> 
> Thanks!

Hi, @jacob-l 

About your points:
> _I think we do not need these files._  

It was some remains from the older project/solution. I removed now.

> _And do we require database.db file in the repository?_  

It is not required, but for convenience I put this file with pre generated Membership tables, so we (and the users) can just F5 and directly test the sample. The most "plug and play" possible experience 😊 , or else we will need 1) Create a new Sqlite Database File 2) copy it to the databases folder 3) execute the included script (InstallMembership.sql)

> _Also, can we show a simple HTML page when the user tries to run the server-side project?_   

Done!

**Comment by FabioSilvaOS** [comment]
> Hi @FabioSilvaOS , we had a chat with Giovanni. He suggests adding something like To-Do items / Tasks to this example. So the user can register, login, and then do something else: see/manage his tasks. We would like to get a more solid Business Template example. What are your thoughts?

I think this is a great idea, I will work on this. We can even put some page visible only logged/registered users, then this functionality can be tested too.

**Comment by FabioSilvaOS** [comment]
The **updated** code from this PR was published in [https://github.com/OpenSilver/OpenSilverBusinessApplication](https://github.com/OpenSilver/OpenSilverBusinessApplication) , with most of the comments addressed, so I'm closing it for now.

**Pull request closed without merging**